acceptance/acceptance_test.go

```diff
@@ -286,6 +288,21 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	cmd.Dir = tmpDir
 	err = cmd.Run()
 
+	// Wait for the files to appear before starting assertion on the output.
+	// This is useful for concurrency control when the CLI spawns subprocesses.
+	missingFiles := config.EventuallyFiles
+	assert.Eventually(t, func() bool {
```

Comment:
Please, let's avoid Eventually(). This makes reasoning about the test very complicated.

For your subprocesses test you can write a bash/python helper that finds the subprocess and waits for it to finish.

Reply:
How do you recommend finding the subprocess? Using it's name?

Reply:
Discussed this option offline: add --telemetry-pidfile that would store the pidfile of spawned telemetry process and wait for it in the script.

To clarify, the race condition here is that eventually will block only until file is created but then it maybe written to and read from at the same time.

Reply:
Done, added bash scripts to wait instead.